aspnet/AspNetWebStack
# Use .NET SDK in two test projects

Use .NET SDK in NetCore.Test project
- for now, unable to target `netcoreapp2.1` in NetCore.Test project
  - Microsoft.Net.Http is not compatible
  - System.Net.Http types have different identities, causing compilation failures
- switch to use `dotnet vstest` for any .NET Core assemblies (just one currently)
  - can use `Xunit` target for all .NET Framework assemblies
- make XunitXml.TestLogger available for .NET Core App testing
  - use above test logger in repo-level testing
  - integrate w/ tests that use the `Xunit` target
- don't import from xunit.msbuild.runner when targeting .NET Core; not needed
- add `$(CopyLocalLockFileAssemblies)` to these (now both .NET SDK) test projects
  - xUnit assemblies were not copied (when they should have been) w/o this

Use .NET SDK in Microsoft.TestCommon
- allows reference in .NET SDK test projects w/o duplicating package references
- not sure how NetStandard.Test targeted `netcoreapp2.1` before
